index.html

```diff
@@ -16,8 +16,46 @@
 </head>
 <body>
   <header>
+    <div class="container line-height-large">
+      <div class="row padding-left">
+        <h3 class="font">Code Journal</h3>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="background-gray height-full">
+      <div data-view="entry-form" class="container">
+        <h2 class="column-full header-spacing">New Entry</h2>
```

Comment:
You should never have a `column` element immediately inside a `container` element. You should make an element with the `row` class that is a child of this `container` class element, and move this `column` element into that `row`